sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/AvroUtils.java

```diff
@@ -887,8 +925,6 @@ private static org.apache.avro.Schema getFieldSchema(
         break;
 
       case DATETIME:
-        // TODO: There is a desire to move Beam schema DATETIME to a micros representation. When
```

Comment:
Is this TODO actually resolved?

Reply:
The new logical type is added (MicrosInstant) but the old field type is yet removed (DATETIME). The latter was attempt in #11456 but there was concern of breaking change. The removal of joda.time (DATETIME's input type) is aimed for Beam 3.0: #19215

Will add this back and link to #19215

Reply:
Done.